src/app/index.ts

```diff
@@ -185,16 +190,30 @@ module.exports = class TheiaPlugin extends Base {
 
     configuring() {
         const options = this.options as any;
+        const creationDate = new Date().getFullYear().toString();
         const pluginName = (options.pluginName as string).replace(/\W/g, '-');
         const pluginType = options.pluginType as string;
         const packageName = pluginName + '-plugin';
         const publisher = options.publisher ? options.publisher as string : 'theia';
         const frontendModuleName = `${publisher}_${packageName}`.replace(/\W/g, '_');
+
+        let licenseId = options.license;
+        let header: string;
+        if (!licenseId || TheiaPlugin.SUPPORTED_LICENSES.indexOf(licenseId) === -1) {
+            licenseId = 'none';
```

Comment:
maybe if license is not supported it should log a warning and default to none (or throw an error) instead of ignoring it

Reply:
It defaults to none.
See no point in throwing an error. 
As about warning... I've thought about it, but decided to have behaviour similar to previous. Maybe adding the warning will be better. Will do it.